ompi/attribute/attribute_predefined.c

```diff
@@ -162,30 +161,28 @@ int ompi_attr_create_predefined(void)
     return ret;
 }
 
-
 int ompi_attr_free_predefined(void)
 {
     int ret;
 
-    if (OMPI_SUCCESS != (ret = free_comm(MPI_TAG_UB)) ||
-        OMPI_SUCCESS != (ret = free_comm(MPI_HOST)) ||
-        OMPI_SUCCESS != (ret = free_comm(MPI_IO)) ||
-        OMPI_SUCCESS != (ret = free_comm(MPI_WTIME_IS_GLOBAL)) ||
-        OMPI_SUCCESS != (ret = free_comm(MPI_APPNUM)) ||
-        OMPI_SUCCESS != (ret = free_comm(MPI_LASTUSEDCODE)) ||
-        OMPI_SUCCESS != (ret = free_comm(MPI_UNIVERSE_SIZE)) ||
-        OMPI_SUCCESS != (ret = free_comm(MPI_FT)) || /* not #if conditional on OPAL_ENABLE_FT_MPI for ABI */
-        OMPI_SUCCESS != (ret = free_win(MPI_WIN_BASE)) ||
-        OMPI_SUCCESS != (ret = free_win(MPI_WIN_SIZE)) ||
-        OMPI_SUCCESS != (ret = free_win(MPI_WIN_DISP_UNIT)) ||
-        OMPI_SUCCESS != (ret = free_win(MPI_WIN_CREATE_FLAVOR)) ||
-        OMPI_SUCCESS != (ret = free_win(MPI_WIN_MODEL))) {
+    if (OMPI_SUCCESS != (ret = free_comm(MPI_TAG_UB)) || OMPI_SUCCESS != (ret = free_comm(MPI_HOST))
+        || OMPI_SUCCESS != (ret = free_comm(MPI_IO))
+        || OMPI_SUCCESS != (ret = free_comm(MPI_WTIME_IS_GLOBAL))
+        || OMPI_SUCCESS != (ret = free_comm(MPI_APPNUM))
+        || OMPI_SUCCESS != (ret = free_comm(MPI_LASTUSEDCODE))
+        || OMPI_SUCCESS != (ret = free_comm(MPI_UNIVERSE_SIZE))
+        || OMPI_SUCCESS != (ret = free_comm(MPI_FT))
+        || /* not #if conditional on OPAL_ENABLE_FT_MPI for ABI */
```

Comment:
Can possibly remove these references to `not #if conditional on OPAL_ENABLE_FT_MPI` now that FT is IN.

Reply:
yes this looks messy.  will remove the comments as part of this PR.

Reply:
I think it's fine. Comment cleanup can be done in another PR.